site/app/views/grading/ElectronicGraderView.php

```diff
@@ -441,6 +441,8 @@ function sortTable(sort_element_index){
      * @return string
      */
     public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_users, $graders, $empty_teams, $show_all_sections_button, $show_import_teams_button, $show_export_teams_button, $show_edit_teams, $past_grade_start_date, $view_all, $sort, $direction, $anon_mode, $overrides, $anon_ids) {
+        $collapsed_sections = isset($_COOKIE['collapsed_sections']) ? explode("\n", $_COOKIE['collapsed_sections']) : [];
```

Comment:
Aren't the cookies supposed to be being used right here on the server side?  The value is then used in `Details.twig`.  I agree that the formatting will need to be changed to remove the newline for it to actually work.  Perhaps I misunderstood what you meant?

Reply:
Oh whoops, I thought this was all new code functionality. That all makes sense now for it to be cookies.

Reply:
Actually it is new functionality but I just skipped that line in my review before it seems.